src/Illuminate/Validation/Rules/RegExp.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Illuminate\Validation\Rules;
+
+use Illuminate\Contracts\Support\Arrayable;
+use Illuminate\Support\Str;
+use Illuminate\Support\Stringable as SupportStringable;
+use Stringable;
+
+class RegExp implements Stringable
+{
+    protected string $regExp;
+    protected bool $negated = false;
+
+    /**
+     * @var string[]
+     */
+    protected array $flags = [];
+
+    /**
+     * Create a new array rule instance.
+     *
+     * @param  string  $regExp
+     * @param  string[]|\Illuminate\Contracts\Support\Arrayable|null  $flags
+     * @return void
+     */
+    public function __construct(string $regExp, $flags = null)
+    {
+        $this->regExp = $regExp;
+        $this->flags(...$flags);
+    }
+
+    public function active(bool $active)
```

Comment:
What is the purpose of this method? Do we even have an `$active` prop? I may just be forgetting what is in `Stringable` though.

Reply:
Good catch! I accidentally copied it from https://github.com/laravel/framework/pull/54519/files#diff-4869bb9b60c679381b374e53caf029cc40da95aaadc62efb7e1e15795489bd90R28-R33